$OpenBSD: patch-async_cabal,v 1.1 2019/09/30 12:00:00 kili Exp $

Relax stm bound. It's already this way upstream, just no release yet.

Index: async.cabal
--- async.cabal.orig
+++ async.cabal
@@ -50,7 +50,7 @@ library
     if impl(ghc>=7.1)
         other-extensions: Trustworthy
     exposed-modules:     Control.Concurrent.Async
-    build-depends:       base >= 4.3 && < 4.12, hashable >= 1.1.1.0 && < 1.3, stm >= 2.2 && < 2.5
+    build-depends:       base >= 4.3 && < 4.13, hashable >= 1.1.1.0 && < 1.3, stm >= 2.2 && < 2.6
 
 test-suite test-async
     default-language: Haskell2010
